assets/emotes.json

```diff
@@ -1,138 +1,130 @@
-{
-    "destiny": [
```

Comment:
Just making sure, is this restructuring necessary and/or okay?

Reply:
yes, the app only uses the `destiny` property